### PR TITLE
Auth relay mutations

### PR DESCRIPTION
**Pull request by pulasthibandara**

### CODE REVIEW

File: .graphqlconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "car-api": {
+      "schemaPath": "schema.graphql",
+      "extensions": {
+        "endpoints": {
+          "dev": "http://localhost:9000/graphql"
+        }
+      }
+    }
+  }
+}
```
Comment (hally9k) [inline .graphqlconfig]: NIce!

File: package.json [hunk 2]
```diff
@@ -63,6 +64,7 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.9.1",
     "get-graphql-schema": "^2.1.1",
```
Comment (hally9k) [inline package.json]: Should we delete graphql-get-schema if we've got the graphql cli in now?

File: src/graphql/mutation/login-mutation.js [hunk 3]
```diff
@@ -3,8 +3,10 @@ import environment from 'environment'
 import { commitMutation, graphql } from 'react-relay'
 
 const mutation: * = graphql`
-    mutation loginMutation($email: String!, $password: String!) {
-        login(email: $email, password: $password)
+    mutation loginMutation($input:  loginMutationInput!) {
+        login(input: $input) {
```
Comment (hally9k) [inline src/graphql/mutation/login-mutation.js]: Tidy, nice one :)

File: src/graphql/mutation/sign-up-mutation.js [hunk 4]
```diff
@@ -4,13 +4,15 @@ import { type SignUpForm } from 'component/auth/sign-up'
...
 const mutation: * = graphql`
-	mutation signUpMutation($authProvider: SignupData!) {
-		signUp(authProvider: $authProvider) {
-			firstName
-			lastName
-			email
-			businessId
-			createdAt
+	mutation signUpMutation($input: signUpMutationInput!) {
+		signUp(input: $input) {
```
Comment (hally9k) [inline src/graphql/mutation/sign-up-mutation.js]: Again this is tidier, shot...

File: package.json (hunk 2 above)
Comment (pulasthibandara) [reply package.json]: Opps, I didn't see it. I'm not sure which one is better.

**Pull request merged**